sdf/deployment.mdx

```diff
@@ -1,75 +1,68 @@
 
-Worker is a process that runs the dataflows continuously.  Unlike ephemeral CLI, worker is long running process.  That means, when you exit CLI, dataflow does not terminate.  It is primary used for running dataflow in production environment.  It can also be used in development environment to run dataflow when there is no need to test package.
+For production deployment, the `deploy` command is used to deploy the dataflow on a worker. All operations available in `run` also apply to deploy, with the following differences:
+- The dataflow is executed on the worker, not within the CLI process. The CLI communicates with the worker on the user's behalf.
+- The dataflow continues running even if the CLI is shut down. It will only terminate if the worker is stopped, shut down, or the dataflow is explicitly stopped or deleted.
+- Dataflows in the worker only have access to published packages, unlike `run` mode, which allows access to local packages. If you need to use a package, you must publish it first.
+- Multiple dataflows can be deployed on the worker, with each dataflow isolated from the others. They do not share any state or memory but can communicate via Fluvio topics.
 
-Worker can run anywhere as long as it can connect to the Fluvio cluster.  Worker can be run in data center, cloud, or edge device.  There is no limit on number of workers that can be run.  Each worker can run multiple dataflows.  It is recommended to a run a single worker per machine.
 
-SDF can target different worker by selecting the worker profile.  Worker profile is associated with the Fluvio profile.  When you switch the Fluvio profile, SDF will automatically switch the worker profile.  Once you have selected the worker, same worker will be used for all dataflow operation until you select a different worker.
+To use deployment mode, it's essential to understand the following concepts:
+- Types of workers
+- Managing workers
+- Deploying dataflows
+- Dataflow lifecycle within a worker
 
-By default, SDF will run the dataflow in the worker.  If you are using InfinyOn Cloud, the worker will be provisioned and automatically registered in the profile. 
+# Worker
 
-# Using Host Worker for Local Cluster
+A worker is the deployment target for a dataflow and must be created and provisioned before deploying a dataflow. The worker can run anywhere as long as it can connect to the same Fluvio cluster. If you're using InfinyOn Cloud, the worker is automatically provisioned.
 
-If you are using local cluster, you need either create `Host` or register the `Remote` worker.  Easiest is to create `Host` worker.  Host worker run in the same machine as the CLI.  
+There is no limit to the number of dataflows you can run on each worker, apart from CPU, memory, and disk constraints. For optimal performance, it is recommended to run a single worker per machine.
 
-To create host worker, you can use the following command:
-```bash
-$> sdf worker create <name>
-```
+## Managing workers
```

Comment:
This section should be moved below "Types of Workers" to match the above Table of Contents